### PR TITLE
Issue/#852

### PR DESCRIPTION
**Pull request by svenvandescheur**
Integreren objecten in search

### CODE REVIEW

File: frontend/zac-ui/libs/features/search/src/lib/search-form/object-search-form/zaak-object-search-form.component.ts [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  /** @type {FieldConfiguration[] Form configuration. */
+  form: FieldConfiguration[] = [
+    {
+      choices: OBJECT_SEARCH_GEOMETRY_CHOICES,
+      label: 'Gebied',
+      name: 'geometry',
+      required: true,
+      value: OBJECT_SEARCH_GEOMETRY_CHOICES[0].value,
+    },
+    {
```
Comment (kelvincy) [inline frontend/zac-ui/libs/features/search/src/lib/search-form/object-search-form/zaak-object-search-form.component.ts:68]: Can we make 2 separate input fields for `Adres` and `Type`? I think asking input this way makes it prone to errors.

Also, is it possible to disallow free form input for `Type` by creating a dropdown with `/api/core/objecttypes`?

Comment (kelvincy) [reply frontend/zac-ui/libs/features/search/src/lib/search-form/object-search-form/zaak-object-search-form.component.ts:68]: When I search on `adres: utrechtsestraat 41` with lower case "u", it is not returning any results. But with an uppercase it does. Is this expected behaviour?

File: frontend/zac-ui/libs/features/search/src/lib/search-form/search-form.component.html [hunk 2]
```diff
@@ -1,123 +1,19 @@
...
-                          [control]="eigenschapnaam"
-                          [searchable]="true"
-                          [items]="zaaktypeEigenschappenData"
-                          [multiple]="false"
-                          bindLabel="name"
-                          bindValue="name"
-                          label="Eigenschapnaam">
-          </gu-multiselect>
+<div class="row">
+  <div class="col-6">
```
Comment (kelvincy) [inline frontend/zac-ui/libs/features/search/src/lib/search-form/search-form.component.html:2]: Doesn't scale on mobile

File: frontend/zac-ui/libs/features/search/src/lib/search-form/search-form.component.html [hunk 3]
```diff
@@ -1,123 +1,19 @@
...
+    <gu-tab-group>
+      <!-- Case (zaak) by property. -->
+      <gu-tab label="Zoeken op eigenschap">
+        <div class="mt-4">
+          <gu-property-search-form [sortData]="sortData" (loadResult)="loadResult.emit($event)"></gu-property-search-form>
```
Comment (kelvincy) [inline frontend/zac-ui/libs/features/search/src/lib/search-form/search-form.component.html:7]: The body of the tab group is not scaling in height when a `multiselect` component is opened

**Review by kelvincy — CHANGES_REQUESTED** [review]
Nice work, looks quite complex to fit the API. I have added a few small comments

File: frontend/zac-ui/libs/features/search/src/lib/search-form/object-search-form/zaak-object-search-form.component.html [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<gu-form [form]="form" buttonLabel="Zoeken" (formSubmit)="submitForm($event)"></gu-form>
+<mat-action-list *ngIf="showZaakObjecten">
+  <mat-list-item *ngFor="let zaakObject of zaakObjects" (click)="selectZaakObject($event, zaakObject)">
```
Comment (kelvincy) [inline frontend/zac-ui/libs/features/search/src/lib/search-form/object-search-form/zaak-object-search-form.component.html:3]: Can you make this more readable? When there are multiple results, there's not so much space between the items .

Screenshot 2021-08-23 at 11 31 08

File: frontend/zac-ui/libs/features/search/src/lib/search-form/object-search-form/zaak-object-search-form.component.ts (hunk 1 above)
Comment (svenvandescheur) [reply frontend/zac-ui/libs/features/search/src/lib/search-form/object-search-form/zaak-object-search-form.component.ts:68]: Yes, the API's only support very limited search. Also, we don't know the actual fields that are available on the objects. I wan't to replace the query field with a free text search after: https://github.com/maykinmedia/objects-api/issues/235 is resolved.

File: frontend/zac-ui/libs/features/search/src/lib/search-form/search-form.component.html (hunk 2 above)
Comment (svenvandescheur) [reply frontend/zac-ui/libs/features/search/src/lib/search-form/search-form.component.html:2]: - [ ] Fix mobile scaling

File: frontend/zac-ui/libs/features/search/src/lib/search-form/search-form.component.html (hunk 3 above)
Comment (svenvandescheur) [reply frontend/zac-ui/libs/features/search/src/lib/search-form/search-form.component.html:7]: - [ ] Investigate auto scaling of tab group.

File: frontend/zac-ui/libs/features/search/src/lib/search-form/object-search-form/zaak-object-search-form.component.html (hunk 4 above)
Comment (svenvandescheur) [reply frontend/zac-ui/libs/features/search/src/lib/search-form/object-search-form/zaak-object-search-form.component.html:3]: - [ ] Improve readability of objects results.

**Review by kelvincy — APPROVED** [review]

**Pull request merged**